src/lib/divider/divider.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {
+  ChangeDetectionStrategy, Component,
+  Input, ViewEncapsulation,
+} from '@angular/core';
+import {coerceBooleanProperty} from '@angular/cdk/coercion';
+
+@Component({
+  moduleId: module.id,
+  selector: 'mat-divider',
+  host: {
+    'role': 'separator',
+    '[attr.aria-orientation]': 'vertical ? "vertical" : "horizontal"',
+    '[class.mat-divider-vertical]': 'vertical',
+    '[class.mat-divider-inset]': 'inset',
+    'class': 'mat-divider'
+  },
+  template: '',
+  styleUrls: ['divider.css'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  preserveWhitespaces: false,
+})
+export class MatDivider {
+  /** Whether the divider is vertically aligned. */
+  @Input() get vertical(): boolean { return this._vertical; }
+  set vertical(value: boolean) {
```

Comment:
for both setter methods, you should be able to move them to a single line which might be a bit more readable.

Reply:
Fixed